review/tests.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from .models import Review, DayCare, DogOwner
+from django.core.validators import ValidationError
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+REVIEW = 'sample review'
+RATING = 5
+DAY_CARE_ID = 1
+DOG_OWNER_ID = 1
+
+
+@pytest.mark.django_db
+@pytest.fixture
+def test_review():
+    return Review.create(review=REVIEW, rating=RATING, daycare_id=DAY_CARE_ID, dogowner_id=DOG_OWNER_ID)
+
+
+@pytest.mark.django_db
+class TestReviewModel:
+    def test_persist_review(self, test_review):
+        assert test_review.review == REVIEW
+        assert test_review.rating == RATING
+        assert test_review.daycare_id == DayCare.objects.get(pk=DAY_CARE_ID)
+        assert test_review.daycare_id_id == DAY_CARE_ID
+        assert test_review.dogowner_id == DogOwner.objects.get(pk=DOG_OWNER_ID)
+        assert test_review.dogowner_id_id == DOG_OWNER_ID
```

Comment:
There is a problem with asserting each class member, if tomorrow the model is changed
this test should be changed as well, which is kind of missing the point in having it.
Also, note that there is no actual logic to check here, its just assigning arguments to the class members.

Reply:
removed the assignment tests, in case we will have logic in the create we will add specific tests for that